pom.xml

```diff
@@ -102,6 +103,20 @@
     <module>ebean-spring-txn</module>
   </modules>
 
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-javadoc-plugin</artifactId>
+          <version>${maven-javadoc-plugin.version}</version>
+          <configuration>
+            <additionalOptions>-Xdoclint:none</additionalOptions>
```

Comment:
The problem is, that there are many javadocs, that uses non-HTML4-compliant tags.
I don't know, what's the best here. Change the HTML5 tags (that obviously work in the browser) to HTML4 tags or disable linting in ebean-paren (or only in specific module)

Reply:
Yes. disable linting seems reasonable for now.